lib/PrintAsObjC/PrintAsObjC.cpp

```diff
@@ -917,6 +915,60 @@ class ObjCPrinter : private DeclVisitor<ObjCPrinter>,
     }
     return hasPrintedAnything;
   }
+    
+  void printRenameForDecl(const AvailableAttr *AvAttr, const Decl *D,
+                          bool includeQuotes) {
+    if (AvAttr->Rename.empty() && isa<ValueDecl>(D))
+      return ;
```

Comment:
We've already checked that this is a ValueDecl at all call sites, so you can probably just drop that. (Also, stray space before the semicolon.)